src/adapters/postgres/user-adapter.ts

```diff
@@ -100,36 +101,36 @@ export class PostgresUserService {
     return results;
   }
 
-  async getUsersDetailsById(userData: Record<string, string>, response: any) {
+  async getUsersDetailsById(userData: any, response: any) {
```

Comment:
@Shubham4026 As discussed check if we can introduce custom Type instead of any

CC @vaivk369